Assets/Editor/FindMissingScriptsRecursively.cs

```diff
@@ -0,0 +1,54 @@
+﻿using UnityEngine;
+using UnityEditor;
+
+public class FindMissingScriptsRecursively : EditorWindow 
```

Comment:
It's for when a warning pops up during the game saying missing script and then clicking it doesn't show you which object it came from.